OrbitGl/CaptureWindow.cpp

```diff
@@ -181,40 +181,13 @@ void CaptureWindow::Pick(int a_X, int a_Y) {
 void CaptureWindow::Pick(PickingID a_PickingID, int a_X, int a_Y) {
   uint32_t type = a_PickingID.m_Type;
   uint32_t id = a_PickingID.m_Id;
```

Comment:
Linux's build complains when there are unused variables.

You can add: 
UNUSED(id);
if you are going to use in the future

Reply:
Is there a way to warn about this when compiling for windows?